pocketcore.rb

```diff
@@ -20,6 +20,6 @@ def install
   end
 
   test do
-    shell_output("#{bin}/pocketcore", "start")
+    shell_output("#{bin}/pocket-core", "start")
```

Comment:
I think the test should be the help command, not the start command.

Reply:
Noted.

Reply:
This should be fixed with the lastest push